Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -45,8 +48,33 @@ ABenchmarkGymGameMode::ABenchmarkGymGameMode()
 
 	NPCSToSpawn = 0;
 	SecondsTillPlayerCheck = 15.0f * 60.0f;
+}
 
-	RNG.Initialize(123456); // Ensure we can do deterministic runs
+void ABenchmarkGymGameMode::GenerateTestScenarioLocations()
+{
+	const float ROAM_RADIUS = 7500.0f;
+	{
+		FRandomStream PlayerStream;
+		PlayerStream.Initialize(123456); // Ensure we can do deterministic runs
+		for (int i = 0; i < ExpectedPlayers; i++)
+		{
+			FVector PointA = PlayerStream.VRand()*ROAM_RADIUS;
+			FVector PointB = PlayerStream.VRand()*ROAM_RADIUS;
+			PointA.Z = PointB.Z = 0.0f;
+			PlayerRunPoints.Push(RunPoints{ PointA, PointB });
```

Comment:
nit - use emplace and just pass the objects directly - ie PlayerRunPoints.Emplace({ PointA, PointB });